nebu/cli/cnxml2html.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+import click
+from cnxdb.triggers.transforms import cnxml_to_full_html
+
+from ._common import common_params
+
+
+@click.command()
+@common_params
+@click.argument('col_path')
+@click.pass_context
+def cnxml2html(ctx, col_path):
+    """Given a collection path COL_PATH, transform all the index.cnxml to
+    index.cnxml.html"""
+
+    for root, dirs, files in os.walk(col_path):
+        if 'index.cnxml' in files:
```

Comment:
Any chance we can stick with using `pathlib` around the codebase? The `<Path>.glob(...)` method will likely do the searching you are walking around for here.

Reply:
So even though the files are already returned in `os.walk`, you want me to use `.glob()` to search for it?  If you confirm that's really what you want, I can change it.  I don't particularly agree with it but I'm not strongly against it.